packages/world_countries/CHANGELOG.md

```
@@ -1,3 +1,7 @@
+## 3.9.0
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)